docs/quickstart/send-slack-alerts.mdx

```diff
@@ -119,8 +119,9 @@ vars:
 ## Alert on source freshness failures
 
 By default, alerts are sent on dbt tests, models and snapshots.
-You can also monitor source freshness by running `dbt run-operation upload_source_freshness` right after each execution of `dbt source freshness`.
+You can also monitor source freshness by running `edr run-operation upload-source-freshness` right after each execution of `dbt source freshness`.
```

Comment:
I think we should specify here that the two commands need to run from the same computer and from the same folder 
(that was also true before but maybe less obvious when we ask to run edr)

Reply:
Definitely agree!